Documentation/proposals/app-level-api.md

```diff
@@ -19,6 +19,15 @@ aforementioned stage1 entrypoints.
 
 The proposed app-level commands are described below.
 
+## `rkt app init`
+Initializes an empty pod having no applications. This returns a single line
+containing the `pod-uuid` which can be used to perform application
```

Comment:
If the command returns immediately after printing the `pod-uuid`, where does it start the pod processes? Would it create a systemd unit in the background?